_posts/2023-11-20-curvature_and_remeshing.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: "New in CGAL: Interpolated Corrected Curvature Measures and Isotropic Remeshing with Sizing Field"
+description: ""
+category:
+tags: [""]
+---
+{% include JB/setup %}
+
+<p>
+The CGAL project has been participating to 12 editions of the <a hef="https://summerofcode.withgoogle.com/">Google Summer of Code</a>
```

Comment:
this link does not work on [this generated page](https://cgal.github.io/cgal-web/2023/11/20/curvature_and_remeshing/)